lib/feature/home_page/home_page.dart

```diff
@@ -0,0 +1,150 @@
+                          ),
+                        ],
+                      ),
+                      child: Column(children: [
+                        const SizedBox(height: 8),
+                        Text(
+                          '${pokemon?[index].name}',
+                          style: const TextStyle(color: Colors.black),
+                          textAlign: TextAlign.left,
+                        ),
+                        Row(
+                          children: [
+                            Column(children: [
+                              Padding(
+                                padding: const EdgeInsets.fromLTRB(20, 1, 1, 1),
+                                child: Text(
+                                    '${pokemon![index].types![0].type!.name}'),
+                              ),
+                              Padding(
+                                padding: const EdgeInsets.fromLTRB(20, 1, 1, 1),
+                                child: Visibility(
+                                  visible: true,
+                                  child: Center(
+                                    child: Text(
+                                        '${typeCheck(pokemon?[index].types!.last.type?.name, pokemon![index].types![0].type!.name) ?? ''}'),
+                                  ),
+                                ),
+                              ),
+                            ]),
+                            Container(
+                              width: 80,
+                              height: 100,
+                              alignment: Alignment.centerRight,
+                              child: Image.network(
+                                'https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/${pokemon?[index].id}.png', // this image doesn't exist
+                                fit: BoxFit.cover,
+                                errorBuilder: (context, error, stackTrace) {
+                                  return Container(
+                                    color: Colors.amber,
+                                    alignment: Alignment.center,
+                                    child: const Text(
+                                      '',
+                                      style: TextStyle(fontSize: 30),
+                                    ),
+                                  );
+                                },
+                              ),
+                            )
+                          ],
+                        ),
+                      ]),
+                    ),
+                    onTap: () {},
+                  ),
+                );
+              },
+            ),
+          ),
+          floatingActionButton: FloatingActionButton(
+            onPressed: loadMore,
+            tooltip: 'Increment',
+            child: const Icon(Icons.circle),
+          ),
+        ));
+  }
+}
+
+typeCheck(String? name, String? name2) {
```

Comment:
What's the reasoning for this?

Reply:
Not familiar with null checker i.e. (" ${album?.title ?? ' '} " )